### ISSUE TITLE
imgui.core.ImGuiError: ImGui assertion error (draw_list->_VtxCurrentIdx < (1 << 16) && "Too many vertices in ImDrawList using 16-bit indices

### ISSUE THREAD

**Issue by mika-fischer** [swistakm/pyimgui#138]
I'm having the issue that the window draw list grows too large for the index type used (unsigned short).

See https://github.com/ocornut/imgui/issues/1188 and https://github.com/ocornut/imgui/blob/e0cab5664adf02c030f9cf2a05d2c334638a85f8/imgui.cpp#L3374-L3383

This leads to the following error:
```
imgui.core.ImGuiError: ImGui assertion error (draw_list->_VtxCurrentIdx < (1 << 16) && "Too many vertices in ImDrawList using 16-bit indices. Read comment above") at imgui-cpp/imgui.cpp:3383
```

It's hard for me to work around this using BeginChild/EndChild because I *want* to draw all these things in the same window...

Therefore I would like to ask if you would be willing to change the `ImDrawIdx` type to `unsigned int` as suggested in the comment (second link). If so I'd be willing to prepare a pull request.

**Comment by swistakm** [comment]
Yeah, why not. If we just make sure that existing integrations work that's not a problem for me.

**Comment by krikkiteer** [comment]
any updates on this ? i'm having the same issue and don't really know how to work around it ;(
any help would be gladly appreciated..

**Comment by swistakm** [comment]
Feature released in 1.2.0
